gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+[versions]
+bytebuddy = "1.12.10"
+compose-compiler = "1.3.0"
```

Comment:
```suggestion
composeCompiler = "1.3.0"
```

Reply:
see https://github.com/cashapp/paparazzi/pull/571#discussion_r971296837

> I went with compose-compiler to match style of other libs/versions, because there's a [ton of compose dependencies](https://mvnrepository.com/artifact/androidx.compose) and this would be the only one with camel-case in this project, everything else has dash separators. It fits in nicely if you look holistically.